cmd/extensions/main.go

```diff
@@ -150,7 +150,7 @@ func main() {
 	agonesInformerFactory := externalversions.NewSharedInformerFactory(agonesClient, defaultResync)
 	kubeInformerFactory := informers.NewSharedInformerFactory(kubeClient, defaultResync)
 
-	server := &httpServer{}
+	server := &httpserver.Server{Logger: logger}
```

Comment:
```suggestion
	server := &http.Server{Logger: logger}
```

That wouldn't be redundant 😃